app/models/ownership.rb

```diff
@@ -4,16 +4,7 @@ class Ownership < ActiveRecord::Base
 
   validates :user_id, uniqueness: { scope: :rubygem_id }
 
-  before_destroy :keep_last_owner
-
-  private
-
-  def keep_last_owner
-    if rubygem.owners.count == 1
-      errors[:base] << "Can't delete last owner of a gem."
-      false
-    else
-      true
-    end
+  def safe_destroy
+    rubygem.owners.count != 1 && destroy ? true : false
```

Comment:
What about to implement this as `before_destroy` callback and keep calling `destroy`?